pkgs/top-level/all-packages.nix

```diff
@@ -330,6 +330,8 @@ with pkgs;
 
   avro-tools = callPackage ../development/tools/avro-tools { };
 
+  avr-sim = callPackage ../by-name/av/avr-sim/package.nix { };
```

Comment:
This line in all-packagrs.nix can be removed thanks to the pkgd/by-name magic.

Reply:
Removed the line.